docs/content/en/docs/whatsnew/changelog.md

```diff
@@ -24,13 +24,55 @@ description: >
   * [Upgrading an EKS Anywhere cluster]({{< relref "../clustermgmt/cluster-upgrades" >}})
-* EKS Anywhere version `v0.19.4` introduced a regression in the Curated Packages workflow due to a bug in the associated Packages controller version (`v0.4.2`). This will be fixed in the next patch release.
 {{% /alert %}}
 
 {{% alert title="General Information" color="info" %}}
 * When upgrading to a new minor version, a new OS image must be created using the new image-builder CLI pertaining to that release.
 {{% /alert %}}
 
+## [v0.20.0](https://github.com/aws/eks-anywhere/releases/tag/v0.20.0)
+
+### Supported OS version details
+|                     | vSphere | Bare Metal | Nutanix | CloudStack | Snow |
+|:-------------------:|:-------:|:----------:|:-------:|:----------:|:----:|
+|    Ubuntu 20.04     |    ✔    |     ✔      |    ✔    |     —      |  ✔   |
+|    Ubuntu 22.04     |    ✔    |     ✔      |    ✔    |     —      |  —   |
+| Bottlerocket 1.20.0 |    ✔    |     -      |    —    |     —      |  —   |
+| Bottlerocket 1.19.4 |    -    |     \*      |    —    |     —      |  —   |
+|      RHEL 8.x       |    ✔    |     ✔      |    ✔    |     ✔      |  —   |
+|      RHEL 9.x       |    —    |     ✔      |    ✔    |     ✔      |  —   |
+
+\* [EKS Anywhere issue regarding deprecation of Bottlerocket bare metal variants](https://github.com/aws/eks-anywhere/issues/7754)
+
+### Added
+- Support for Kubernetes v1.30
+- Support for configuring api server flags in cluster spec ([#7755](https://github.com/aws/eks-anywhere/pull/7755))
+- Redhat 9 support for Bare Metal ([#3032](https://github.com/aws/eks-anywhere-build-tooling/pull/3032))
+- Support for configuring kubelet settings in cluster spec ([#8265](https://github.com/aws/eks-anywhere/pull/8265))
+- Support for kubelet settings on BottleRocket nodes ([#3266](https://github.com/aws/eks-anywhere-build-tooling/pull/3266))
+- Support for control plane failure domains on Nutanix ([#8192](https://github.com/aws/eks-anywhere/pull/8192))
+
+### Changed
+- Generate cluster config command includes OSImageURL in tinkerbell machine config objects ([#8226](https://github.com/aws/eks-anywhere/pull/8226))
+- Added EKS-D for 1-30:
+  - [`v1-30-eks-7`](https://distro.eks.amazonaws.com/releases/1-30/7/)
+- Cilium: `v1.13.9` to `v1.13.16`
+- Cluster API: `v1.6.1` to `v1.7.2`
+- Cluster API Provider vSphere: `v1.8.5` to `v1.10.0`
+- Cluster API Provider Nutanix: `v1.2.3` to `v1.3.1`
+- Flux: `v2.2.3` to `v2.3.0`
+- Kube-vip: `v0.7.0` to `v0.8.0`
+- Image-builder: `v0.1.24` to `v0.1.26`
+- Kind: `v0.22.0` to `v0.23.0`
+- Etcdadm Controller: `v1.0.17` to `v1.0.21`
+
+### Removed
+- Support for Kubernetes v1.25
+- Support for certain curated packages CLI commands ([#8240](https://github.com/aws/eks-anywhere/pull/8240))
+
+### Fixed
+- CLI commands for packages to honor the registry mirror setup in cluster spec ([#8026](https://github.com/aws/eks-anywhere/pull/8026))
+
```

Comment:
```suggestion
- Cluster status reconciliation for worker nodes when cluster autoscaler is configured ([8254](https://github.com/aws/eks-anywhere/pull/8254))

```

Reply:
this is already in [release-0.19](https://github.com/aws/eks-anywhere/pull/8283) ?